face/package.json

```diff
@@ -55,7 +55,8 @@
     "backbone-decorators": "0.0.4",
     "backbone.marionette": "^3.0.0",
     "backbone.wreqr": "^1.3.0",
-    "d3": "^3.5.12",
+    "d3": "^4.2.7",
```

Comment:
you never use the `d3` module itself.

You need to explcitly put dependency for each module, like `d3-selection`, `d3-axis` and so on.